test/DaprExtensionTests/Tests/DaprTopicTriggerTests.cs

```diff
@@ -60,9 +61,10 @@ public async Task TopicNameInAttributeAsBindingExpression()
         {
             int input = 42;
 
-            // The method name is DotNetMethodName
-            // The function name is FunctionName
+            // The method name is DotNetBindingResolution
+            // The function name is DotNetBindingResolution
             // The topic name is MyTopic
```

Comment:
This does not match:-
public static void DotNetBindingResolution(
                [DaprTopicTrigger("%PubSubName%", Topic = "%TopicName%")] int input,
                ILogger log) => log.LogInformation(input.ToString());

Reply:
fixed.